core-rust/core-api-server/core-api-schema.yaml

```diff
@@ -3263,20 +3309,25 @@ components:
             This hash is `ACC_{N+1} = Blake2b-256(CONCAT(ACC_N, LEDGER_HASH_{N}))`, starting with `ACC_0 = 000..000` the pre-genesis accumulator.
         status:
           $ref: "#/components/schemas/LtsCommittedTransactionStatus"
-        fungible_outcome:
-          description: A list containing all account fungible balance updates
+        fungible_entity_balance_changes:
+          description: A list containing all account fungible sbalance updates
```

Comment:
Typo in balance, and it's not just accounts any more.

Perhaps the following:
`A list of all fungible balance updates which occurred in this transaction, aggregated by the global entity (such as account) which owns the vaults which were updated.`